tests/data/pia.asc

```diff
@@ -0,0 +1,38 @@
+;	V3 text file written for MicroBrightField products.
```

Comment:
have you run this over the full collection of ASC morphologies? Did the expected marker count match?

Reply:
I have checked a few morphologies but I don't know what is the expected number of markers in all the morphologies we have.

Reply:
Could you grep for the markers, count the number of hits, and compare that to how many are found by the code?

Reply:
Markers are not easily greppable. They are just sexp that do not have a (Axon), (Cellbody) or (Dendrite) in the header. I can try to grep for strings and filter out the one we know (ie "CellBody") and see if it matches.

Reply:
I just spot checked a couple files, but they seem to have a comment saying ';  End of markers', so I thought that might work to grep on?

Reply:
Not necessarily. I've run the following code to make sure that all markers where being found in our morphology repository. Markers are either top level sexps starting with a quote (except Cell Body quotes), or supported NeuroLucida markers. 

```python
from pathlib import Path
import yaml
import re

from tqdm import tqdm
from morphio import Morphology, set_maximum_warnings
from numpy.testing import assert_equal

def is_marker(line):
    return any((re.match('\("', line) and '"CellBody' not in line and '"Cell Body' not in line,
                re.match('\((Dot|FilledCircle|SnowFlake|Asterisk|OpenCircle|OpenStar|Flower|OpenSquare|FilledStar|DoubleCircle)', line)))

def test_markers():
    set_maximum_warnings(0)
    folder = '/home/bcoste/workspace/MorphologyRepository'
    paths = list(Path(folder).rglob('*.ASC'))
    for p in tqdm(paths):
        if p.name in {'cr161027_A_idA.ASC'}:
            continue
        cell = Morphology(p)
        lines = list(filter(is_marker, open(p).readlines()))

        s = [marker.label for marker in cell.markers]
        msg = f'In {p}\n{s}\n VS \n{lines}'
        assert_equal(len(cell.markers), len(lines), msg)
```